web-frame/src/app-shell/default-shell/default-shell.component.scss

```diff
@@ -1,6 +1,29 @@
 /**
  * @license
- * Copyright (c) 2018 THREEANGLE SOFTWARE SOLUTIONS SRL
+ * Copyright (c) 2020 THREEANGLE SOFTWARE SOLUTIONS SRL
  * Available under MIT license webFrame/LICENSE
  */
 
+app-default-shell {
+  display: grid;
+  grid-auto-flow: row;
+  grid-template-rows: min-content 1fr;
+  min-height: 100vh;
```

Comment:
FYI: Currently, the Default and Minimal shell styles are basically the same and I've copy/pasted the CSS, but in a real application this will not be the case and the styles will diverge.